### PR TITLE
Add CXF and ES platform BOM overrides

### PR DESCRIPTION
**Pull request by Croway**

### CODE REVIEW

File: tooling/redhat-camel-spring-boot-bom/pom.xml [hunk 1]
```diff
@@ -32,14 +32,16 @@
...
-	<!-- Camel / SpringBoot dependency versions -->
+        <!-- Camel / SpringBoot dependency versions -->
         <version.camel>3.20.1.redhat-00005</version.camel>
         <version.camel-spring-boot>3.20.1.redhat-00012</version.camel-spring-boot>
         <version.spring-boot>2.7.4</version.spring-boot>
 
         <!-- Third party dependency versions -->
         <version.snakeyaml>1.33</version.snakeyaml>
         <version.hsqldb>2.7.1</version.hsqldb>
+        <version.elasticsearch-java-client>8.5.2</version.elasticsearch-client>
```
Comment (mcarlett) [inline tooling/redhat-camel-spring-boot-bom/pom.xml]: please fix typo `<version.elasticsearch-java-client>8.5.2</version.elasticsearch-client>` to `<version.elasticsearch-java-client>8.5.2</version.elasticsearch-java-client>` (the close tag)

**Comment by mcarlett** [comment]
looking at the effective pom I see
```
      <dependency>
        <groupId>org.elasticsearch</groupId>
        <artifactId>elasticsearch</artifactId>
        <version>7.17.6</version>
      </dependency>
      <dependency>
        <groupId>org.elasticsearch.client</groupId>
        <artifactId>transport</artifactId>
        <version>7.17.6</version>
      </dependency>
      <dependency>
        <groupId>org.elasticsearch.client</groupId>
        <artifactId>elasticsearch-rest-high-level-client</artifactId>
        <version>7.17.6</version>
      </dependency>
      <dependency>
        <groupId>org.elasticsearch.distribution.integ-test-zip</groupId>
        <artifactId>elasticsearch</artifactId>
        <version>7.17.6</version>
        <type>zip</type>
      </dependency>
      <dependency>
        <groupId>org.elasticsearch.plugin</groupId>
        <artifactId>transport-netty4-client</artifactId>
        <version>7.17.6</version>
      </dependency>
```
do you think we can manage those versions too?

**Comment by Croway** [comment]
> looking at the effective pom I see
> 
> ```
>       <dependency>
>         <groupId>org.elasticsearch</groupId>
>         <artifactId>elasticsearch</artifactId>
>         <version>7.17.6</version>
>       </dependency>
>       <dependency>
>         <groupId>org.elasticsearch.client</groupId>
>         <artifactId>transport</artifactId>
>         <version>7.17.6</version>
>       </dependency>
>       <dependency>
>         <groupId>org.elasticsearch.client</groupId>
>         <artifactId>elasticsearch-rest-high-level-client</artifactId>
>         <version>7.17.6</version>
>       </dependency>
>       <dependency>
>         <groupId>org.elasticsearch.distribution.integ-test-zip</groupId>
>         <artifactId>elasticsearch</artifactId>
>         <version>7.17.6</version>
>         <type>zip</type>
>       </dependency>
>       <dependency>
>         <groupId>org.elasticsearch.plugin</groupId>
>         <artifactId>transport-netty4-client</artifactId>
>         <version>7.17.6</version>
>       </dependency>
> ```
> 
> do you think we can manage those versions too?

not needed, these dependencies are used by older/not supported camel-elasticsearch

**Review by mcarlett — APPROVED** [review]

**Review by cunningt — APPROVED** [review]
Looks good to me

**Pull request merged**